code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -4,7 +4,7 @@
 	icon = 'icons/mob/human.dmi'
 	icon_state = "human_basic"
 	appearance_flags = KEEP_TOGETHER|TILE_BOUND|PIXEL_SCALE|LONG_GLIDE
-	hud_possible = list(HEALTH_HUD,STATUS_HUD,ID_HUD,WANTED_HUD,IMPLOYAL_HUD,IMPCHEM_HUD,IMPTRACK_HUD, NANITE_HUD, DIAG_NANITE_FULL_HUD,ANTAG_HUD,GLAND_HUD,SENTIENT_DISEASE_HUD,FAN_HUD) // VoidCrew edit - Nanites
+	hud_possible = list(HEALTH_HUD,STATUS_HUD,ID_HUD,WANTED_HUD,IMPLOYAL_HUD,IMPCHEM_HUD,IMPTRACK_HUD, NANITE_HUD, DIAG_NANITE_FULL_HUD,ANTAG_HUD,GLAND_HUD,SENTIENT_DISEASE_HUD,FAN_HUD,FACTION_HUD) // VoidCrew edit - Nanites
```

Comment:
This should too, what you should do is:
```suggestion
/mob/living/carbon/human/Initialize()
    . = ..()
    hud_possible |= list(NANITE_HUD, DIAG_NANITE_FULL_HUD,ANTAG_HUD,FACTION_HUD)
```

inside of a voidcrew file of the same name, this way we can keep it separate and not even have to touch this list (also do the nanites too pls because willard was lazy and didn't do them <3)

Reply:
i get everything here except the antag hud does nanites use them as they aren't called for my faction hud

Reply:
also it seems that adding to the list on Initialize() isn't adding them properly to the list as those huds stop working